### ISSUE TITLE
Add filters to disable currency explicit formatting

### ISSUE THREAD

**Issue by jessepearson** [Automattic/woocommerce-payments#3514]
### Description
Presently all merchants will see explicit formatting in the admin even if Multi-Currency is disabled, or if only one currency is enabled. Some merchants may not like this, so we should add a filter to allow disabling explicit formatting.

**Comment by vbelolapotkov** [comment]
> Some merchants may not like this, so we should add a filter to allow disabling explicit formatting.

@jessepearson is there any discussion with product or dev team about this change? I don't have a full context for this but from the first look it seems to be a hypothetical case.

**Comment by jessepearson** [comment]
This stemmed from this forum thread: https://wordpress.org/support/topic/usd-appears-everywhere-now/

The merchant does not like that explicit formatting now shows when they are only using a single currency. Explicit formatting may be a weird thing for merchants that are only dealing in a single currency, and they may want to hide it.

Explicit formatting is something we chose to add, even though it is not needed in all instances. Adding a filter will allow merchants to disable it on their end if they choose to do so.

**Comment by thuautp** [comment]
Also requested in 4714637-zen. Here's their suggestion:

> I do not think any shop owner would like to have it this way. My suggestion is to just remove it by default and make a filter to enable it.

> The problem is the way how this string is added is bad as well and I cannot simply remove it using CSS. I would have to wrap it with some additional HTML using JS and then disable, which is a really dirty hack and I would not like to do that. Maybe at least you could wrap this string with some html tags with a class, so it could be removed with a simple css.

**Comment by vbelolapotkov** [comment]
Looks like there might be filter, html/css or settings option approach.

cc @LevinMedia for additional thoughts/input on this.

**Comment by LevinMedia** [comment]
>Explicit formatting is something we chose to add, even though it is not needed in all instances. Adding a filter will allow merchants to disable it on their end if they choose to do so.

we should only be displaying explicit formatting once  merchant has added another currency beyond their store currency. We could even take it a step further and only display explicit formats once they've added two currencies with the same symbol. 

IE: my default currency is USD, I add CAD - we show explicit format. 
My default currency is USD, I add GBP - we do not need to show explicit format.

**Comment by jessepearson** [comment]
> we should only be displaying explicit formatting once merchant has added another currency beyond their store currency. We could even take it a step further and only display explicit formats once they've added two currencies with the same symbol.

If I recall correctly the current status of the formatting was based upon this Slack conversation: p1628089650324900-slack-C01B8KNUYSW . This was the final outcome:

> In order to resolve this - lets ignore, or not apply any CLDR formatting to the store's default currency. This should only be the case on the front end of a store (customer facing prices) The only exception to this rule would be the addition of the currency code (explicit format) to things like the order totals in the cart / checkout.  The addition of the explicit format should only happen if merchant is using the multi currency feature (That specific case will be resolved by the issue @bubbles is working on for the patch release. )
> We should continue to use CLDR formatting for all prices through out the admin for consistency and clarity.

I am not sure how hard it would be to disable explicit formatting in the admin if additional currencies are not enabled, this is something we would have to research. However, I do believe that we should either have explicit on or off, not alternate if it's used or not like in the example mentioned:

> IE: my default currency is USD, I add CAD - we show explicit format.
> My default currency is USD, I add GBP - we do not need to show explicit format.

I feel like only showing explicit format when the currency symbols match could lead to inconsistent layouts in lists of orders, transactions, etc.

**Comment by LevinMedia** [comment]
@jessepearson I read back through the original forum ask, and it appears what the OP was asking for was for explicit formats NOT to appear on anything that their customer sees - When I replied to the GH issue yesterday, I made the false assumption that we were still showing explicit formats everywhere on the frontend when WCpay was activated and enabled. However,  I tested the experience today, and noted that explicit format isn't appearing anywhere on the front end for a customer, including in confirmation emails, and on the order confirmation screen. This changes once a second currency is added via Multi-currency. 
 
So with that as the current behavior (and what I would describe as the expected behavior) it seems at least now on the front end, that this issue has been resolved and we can close it out.

I've created a new issue ( #3793 ) to hide explicit formats from the admin too if they're only using the default currency. Then that way the admin behavior end matches the front end.
